### PR TITLE
kdeconnect: add package

### PR DESCRIPTION
**Pull request by Biswa96**
I have used the ucrt64 kdeconnect package in Windows 10 with an Android phone.

<details><summary>Click here to see the screenshot of kdeconnect</summary>

image

</details>

### CODE REVIEW

File: mingw-w64-kdeconnect/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+build() {
+  declare -a _extra_config
+  if check_option "debug" "n"; then
+    _extra_config+=("-DCMAKE_BUILD_TYPE=Release")
+  else
+    _extra_config+=("-DCMAKE_BUILD_TYPE=Debug")
+  fi
+
+  _kde_build_env
+  CXXFLAGS+=" -std=c++20 -Wno-c++11-narrowing" \
```
Comment (ognevny) [inline mingw-w64-kdeconnect/PKGBUILD:101]: meson has cpp_std option. `-Dcpp_std=c++20` doesn't work?

Comment (Biswa96) [reply mingw-w64-kdeconnect/PKGBUILD:101]: This is cmake btw. I'll add the proper solution in upstream later. There are many things that need to be fixed.

Comment (ognevny) [reply mingw-w64-kdeconnect/PKGBUILD:101]: cmake also should have an option for it, shouldn't it?

Comment (Biswa96) [reply mingw-w64-kdeconnect/PKGBUILD:101]: CMAKE_CXX_STANDARD

Comment (Biswa96) [reply mingw-w64-kdeconnect/PKGBUILD:101]: I have sent a patch upstream https://invent.kde.org/network/kdeconnect-kde/-/merge_requests/729

**Pull request merged**